packages/dds/merge-tree/src/mergeTreeNodes.ts

```diff
@@ -591,7 +591,10 @@ export abstract class BaseSegment implements ISegment {
 
 			case MergeTreeDeltaType.OBLITERATE:
 				const moveInfo: IMoveInfo | undefined = toMoveInfo(this);
-				assert(moveInfo !== undefined, 0x86e /* On obliterate ack, missing move info! */);
+				assert(
+					moveInfo?.movedSeqs !== undefined,
```

Comment:
This assert change should be reverted, instead can non-null assert movedSeqs below.